### ISSUE TITLE
How could you determine the hyperparameter value on the test set?

### ISSUE THREAD

**Issue by Yiming-M** [okankop/Driver-Anomaly-Detection#11]
Hi @okankop ,

I think you might have made an error in the code for validation/testing. It seems that the value of threshold is determined by the results on validation/testing. If you split the original test set into a validation set and a new test set, then it's fine to tune this hyperparameter on the validation set. But the problem is, it seems that you didn't do it...

Many thanks,
Yiming
